spec/game_spec.rb

```diff
@@ -0,0 +1,359 @@
+require './lib/game'
```

Comment:
Loads of progressively tougher tests. Really testing the the robustness of the gaming mechanism. V.Good.